packages/amplication-notifications-service/src/contracts/interfaces/messagePattern.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface IMessagePattern {
+  userId: string;
+  payload: { [key: string]: string };
+  template: string;
```

Comment:
@arielweinberger I remember we talked about some kind of mapping between template and notification name but I am not sure why we need it. For example:
![image](https://user-images.githubusercontent.com/39680385/174996073-2475794d-0e71-42fb-96e7-2be760318529.png)

we can use 'code-view-is-ready' as the template name as well as for the notification name

Reply:
Yes indeed. From a separation of concerns point of view, the publisher should not be concerned about a template ID.

We should use a human-readable name, such as "code-review-is-ready" (I would prefer something like `code_review.ready` so we can cover multiple states in the future.

Then it's up to the Notification Service to handle template names.

Today we use novu, tomorrow we might go back to SendGrid. Specific template IDs/names should be abstracted.